src/App/App.js

```diff
@@ -2,15 +2,17 @@ import React, { Component } from 'react';
 import Header from '../Header/Header';
 import Login from '../Login/Login';
 import Movies from '../Movies/Movies';
+import ShowPage from '../ShowPage/ShowPage';
 import './App.css';
 import { getAllMovies } from '../apiCalls'
 
 class App extends Component {
   constructor() {
     super()
-    this.state = {
+    this.state= {
       movies: [],
       loginData: {},
+      selectedMovieId: 694919,
```

Comment:
I'm assuming this is hardcoded in just for the sake of checking to make sure it works properly?